pet-api/pet-api/Domain/Entities/Product.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace pet_api.Domain.Entities
+{
+   public class Product
+   {
+        public int ProductId { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public double Price { get; set; }
```

Comment:
decimal, all types related to the money should be decimal

Reply:
property type has been changed to Decimal